### ISSUE TITLE
Ask for an advanced math for machine learning book for phd students and researchers

### ISSUE THREAD

**Issue by zhmlcg** [mml-book/mml-book.github.io#585]
As mentioned in the foreword, this book is intended to narrow or even close the gap between high school mathematics and the college machine learning textbook. However, the same large gap exists between college machine learning textbook and mathematics level required to conduct research on machine learning (as well as deep learning, reinforcement learning, etc.). As with the common complaint of students of mathematics around the globe that the topics covered seem to have little relevance to practical problems, a concern I have is that the math in this book is too simple to be of any practical relevance to actual research problems. Phd students or researchers on machine learning are longing for such a book to equip themselves with necessary math background to survive the phd program or challenging math problems encountered. Can you please write an advanced and deeper math for machine learning book for those poor phd students and researchers? Thanks a million in advance.

**Comment by mpd37** [comment]
We do believe that the applied chapters are actually relevant in practice. Maybe not so much for PhD-level research, but for people applying machine learning. But we definitely agree that this is an introductory book. 
I (and I believe I can say "we") will not be writing an advanced book any time soon: first, writing this 'simple' book took three years, and we don't have the time or energy at the moment. Second, it is impossible to agree what should go into an "advanced" book (and to what depth). You can easily write thousands of pages on measure theory, functional analysis, topology, optimization, numerics, learning theory, approximate inference etc. and you would still miss out topics some people care about. Therefore, for these advanced topics, you will need to read an specialized book.

To summarize: Agree that this book is only an introduction, but I don't see any possibility of writing a general book on advanced mathematics for machine learning.
